injector/grpc_disruption_test.go

```diff
@@ -0,0 +1,134 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package injector_test
+
+import (
+	"testing"
+
+	"github.com/DataDog/chaos-controller/api/v1beta1"
+	"github.com/DataDog/chaos-controller/grpc"
+	. "github.com/DataDog/chaos-controller/injector"
+
+	pb "github.com/DataDog/chaos-controller/grpc/disruption_listener"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/stretchr/testify/mock"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+func TestSendAndCleaDisruption(t *testing.T) {
```

Comment:
This file should use `injector_suite_test.go` to match the other injector unit tests, rather than importing testing.T itself

Reply:
hm, so I need the variable `t` to run `disruptionListenerClient.AssertExpectations(t)` at the bottom of the file so this is using a slightly different testing approach than the other files by leveraging `AssertExpectations` (there's no equivalent in the `Expect` universe that allows you to express nuances in function calls that the `matchedBy` allows (since we aren't sure which of two maps actually gets returned).

Reply:
This is what I get when I make it a simple `Describe` test `injector/grpc_disruption_test.go:130:48: undefined: t`